ApplyToBecomeInternal/ApplyToBecomeCypressTests/cypress/integration/Record_Decision/103195_create-approved-decision.js

```diff
@@ -45,8 +48,10 @@ describe('103195 Record new Approved decision', () => {
         cy.ApprovedMessageBanner().should('contain.text', 'Decision recorded')
         checkSummary()
         // confirm project status has been updated
-        cy.visit(Cypress.env('url') + '/project-list')
-        cy.projectStateId().should('contain.text', 'APPROVED')
+        RecordDecision.selectProject().then(id => {
```

Comment:
`selectProject()` will go to the project list and select the first project. 
This may slow down the test when you don't need to.
I'd try adding this instead - 
```
cy.url().then(url => {
    const id = RecordDecision.getIdFromUrl(url));
    cy.visit(Cypress.env('url') + '/project-list')
    cy.get(`[id="project-status-${id}"]`).should('contain.text', 'APPROVED WITH CONDITIONS')
};
```

Reply:
I haven't tested this, so may need tweaking.